ACP2E-2419: Document the updated behavior of isEmailAvailable endpoint in 2.4.7-b1 (#198)
AdobeDocs/commerce-php: src/pages/development/backward-incompatible-changes/highlights.md

### `isEmailAvailable` API

> The default behavior of the [`isEmailAvailable`](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/queries/is-email-available/) GraphQL query and ([`V1/customers/isEmailAvailable`](https://adobe-commerce.redoc.ly/2.4.6-admin/tag/customersisEmailAvailable/#operation/PostV1CustomersIsEmailAvailable)) REST endpoint has changed. By default, the API now always returns `true` regardless of provided email address. 

Reviewer: ```suggestion
The default behavior of the [`isEmailAvailable`](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/queries/is-email-available/) GraphQL query and ([`V1/customers/isEmailAvailable`](https://adobe-commerce.redoc.ly/2.4.6-admin/tag/customersisEmailAvailable/#operation/PostV1CustomersIsEmailAvailable)) REST endpoint has changed. By default, the API now always returns `true`.
```

> The new default behaviour also affects the checkout workflow for guests that do not realize they already have an account. Previously, by default, guests that do not realize they already have an account was prompted to sign in after entering their email address at the checkout page. Now, by default, guests are not offered to sing in even if they entered an email that is already belongs to an existing registered customer account.

Reviewer: ```suggestion
The new default behaviour also affects the checkout workflow for guests that do not realize they already have an account. Previously, by default, when a guest supplied an email address that matched an existing customer account, they were prompted to sign in. Now, they are no longer prompted to sign in.
```

> By setting the "Enable Guest Checkout Login" configuration option located at "Config > Sales > Checkout" to "Yes", merchants can enable the original behavior, which is to return `true` if the email does not exist in the database and `false` if it exists, and prompting guest users to sign in during the checkout if their email already belongs to a registered customer account. Setting this option to yes, however, comes at the cost of exposing information to unauthenticated users.

Reviewer: Merchants can restore the original default behavior of the  `isEmailAvailable` API and checkout flow by setting the **Stores** > Configuration > **Sales** > **Checkout** > **Enable Guest Checkout Login** field to **Yes**.   However, doing this can expose customer information to unauthenticated users.

### `isEmailAvailable` API

The default behavior of the [`isEmailAvailable`](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/queries/is-email-available/) GraphQL query and ([`V1/customers/isEmailAvailable`](https://adobe-commerce.redoc.ly/2.4.6-admin/tag/customersisEmailAvailable/#operation/PostV1CustomersIsEmailAvailable)) REST endpoint has changed. By default, the API now always returns `true`.
The new default behaviour also affects the checkout workflow for guests that do not realize they already have an account. Previously, by default, when a guest supplied an email address that matched an existing customer account, they were prompted to sign in. Now, they are no longer prompted to sign in.

> Merchants can restore the original default behavior of the `isEmailAvailable` API and checkout flow by setting the **Stores > Configuration > Sales > Checkout > Enable Guest Checkout Login field** to **Yes**. However, doing this can expose customer information to unauthenticated users.

Reviewer: ```suggestion

Merchants can restore the original default behavior of the `isEmailAvailable` API and checkout flow by setting the **Stores > Configuration > Sales > Checkout > Enable Guest Checkout Login field** to **Yes**. However, doing this can expose customer information to unauthenticated users.

```

This suggestion adds a blank line before and after line 62. Without the blank line after, the PR won't pass linting checks. Sorry I didn't notice it earlier.
